core/services/run_manager.go

```diff
@@ -357,6 +376,7 @@ func (jm *runManager) updateAndTrigger(run *models.JobRun) error {
 		return err
 	}
 	if run.Status == models.RunStatusInProgress {
+		runsResumed.Inc()
```

Comment:
Curious if `runsResumed` will include scenarios like `pending_confirmation`, `pending_sleep`, etc

Reply:
All of the above.